src/components/PcpFeed/PcpCard/index.js

```diff
@@ -0,0 +1,114 @@
+import * as moment from "moment-timezone";
+import PropTypes from "prop-types";
+import { makeStyles } from "tss-react/mui";
+
+const useStyles = makeStyles()((theme) => ({
+	container: {
+		color: theme.palette.text.primary,
+		background: "white",
+		padding: "0.6rem 0 0.6rem 0",
+		display: "flex",
+		gap: "0.625rem",
+		flexDirection: "row",
+		width: "100%",
+		borderBottom: "1px solid #585858",
+	},
+	card: {
+		width: "100%",
+	},
+	header: {
+		fontWeight: 700,
+	},
+	content: {
+		display: "flex",
+		flexDirection: "row",
+		justifyContent: "space-between",
+		flexWrap: "wrap",
+	},
+	status: {
+		display: "inline-flex",
+		marginLeft: "auto",
+		"& div": {
+			fontWeight: 700,
+			padding: "0.4rem 0.875rem 0.4rem 0.875rem",
+			borderRadius: "1rem",
+			fontSize: "0.875rem",
+			lineHeight: "1rem",
+			color: "white",
+			display: "flex",
+			justifyContent: "center",
+			width: "5.624rem",
+		},
+	},
+	closed: {
+		background: "#EF0A0A",
+	},
+	open: {
+		background: "#31A35F",
+	},
+	upcoming: {
+		background: "#FCBA19",
+	},
+}));
+
+const PcpCard = ({ dateCompleted, dateStarted, phaseName, projectName }) => {
+	const { classes } = useStyles();
+
+	const formattedDate = (date) => {
```

Comment:
Please remove this and use `getLongDate` from the date service (coming soon in PR #1).

Reply:
Addressed in c53b5c2